cmd/run.go

```diff
@@ -97,6 +97,7 @@ var runCmd = &cobra.Command{
 		go func() {
 			err := proj.RunServices(localCloud, stopChan, updatesChan)
 			if err != nil {
+				localCloud.Stop()
 				panic(err)
 			}
 		}()
```

Comment:
Can address this in a separate PR, but we might want to actually capture and return the errors here instead of panicking.